Assets/SC_Cube.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SC_Cube : MonoBehaviour
+{
+    private Transform transform;
+
+    public float rotationspeed;
```

Comment:
Public fields should be referred to as PascalCase. However, there is no particular need to make this field public. It could be made private using the [SerializeField] header.
```suggestion
    [SerializeField] private float rotationSpeed;
```